public/sitemap.xml

```diff
@@ -62,7 +62,7 @@
 		<priority>0.64</priority>
 	</url>
 	<url>
-		<loc>https://opentermsarchive.org/fr/press</loc>
+		<loc>https://opentermsarchive.org/fr/media</loc>
```

Comment:
Please also add a permanent redirect in `next.config.js`
https://nextjs.org/docs/api-reference/next.config.js/redirects